runtime/lua/vim/lsp/util.lua

```diff
@@ -1481,6 +1481,24 @@ function M.open_floating_preview(contents, syntax, opts)
   return floating_bufnr, floating_winnr
 end
 
+local function convert_utf_to_byte(line, idx, offset_encoding)
```

Comment:
There's `get_line_byte_from_position` which is similar - is it possible to merge the two?

Reply:
agreed, that makes sense. Also found a TODO regarding `offset_encoding` in that function